src/domain/proof-of-liabilities/create-liabilities.ts

```diff
@@ -0,0 +1,75 @@
+import { LedgerService } from "@services/ledger"
```

Comment:
domain should not depend on services layer, also, this logic should be in app layer

Reply:
Yes, sorry I forgot that part! I will make the change.